### ISSUE TITLE
Is CGLIB 3.3.0 compatible with latest ASM version 9.5?

### ISSUE THREAD

**Issue by sahil3390** [cglib/cglib#216]
As per our information latest CGLIB 3.3.0 is compatible with ASM 7.1, is there any plan to release CGLIB compatible with latest ASM version?

Currently we are using CGLIB 2.2, if we plan to move to latest CGLIB, is there a major change between these releases which may impact migration?

**Comment by sameb** [comment]
No, there are no plans to update cglib.  The library is unmaintained.
